src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -79,12 +82,25 @@
 </kd-help-section>
 
 <kd-help-section>
-  <md-switch ng-model="ctrl.isExternal" class="md-primary"
+  <md-checkbox ng-model="ctrl.isExternal" class="md-primary no-ink"
```

Comment:
md-primary is for blue color. no-ink is for disabling the fancy droplet-like animation on click.